modules/virt-disabling-mac-address-pool-for-namespace-cli.adoc

```diff
@@ -5,15 +5,15 @@
 [id="virt-disabling-mac-address-pool-for-namespace-cli_{context}"]
 = Disabling a MAC address pool for a namespace in the CLI
 
-Disable a MAC address pool for virtual machines in a namespace by removing the `mutatevirtualmachines.kubemacpool.io` label.
+Disable a MAC address pool for virtual machines in a namespace by adding the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace.
 
 .Procedure
 
-* Remove the KubeMacPool label from the namespace.
-The following example removes the KubeMacPool label from two namespaces, `<namespace1>` and `<namespace2>`:
+* Add the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace.
+The following example adds the KubeMacPool label to two namespaces, `<namespace1>` and `<namespace2>`:
```

Comment:
To avoid confusion between KubeMacPool (the service) and the actual label, I would suggest rewording this as "The following example disables KubeMacPool for two namespaces, `<namespace1>` and `<namespace2>`:"